### PR TITLE
add more text on values to CoC

### PR DESCRIPTION
**Pull request by sckott**
I've taken the last sentence of the first paragraph and built on that for a values paragraph. I had a hard time with this text, so I thought I'd just get something down and then we can talk about it;  and it seems important to have input from others rather than just my take. Thoughts on this?

via #17

### CODE REVIEW

File: conduct/code-of-conduct.md [hunk 1]
```diff
@@ -2,7 +2,9 @@ This is a template code of conduct that event organizers can use for conferences
 
 # Code of Conduct
 
-`<EVENT>` is dedicated to providing a harassment-free experience for participants at our event(s). `<EVENT>` is a conference intended for professional networking and collaboration in the R community. We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. 
+`<EVENT>` is dedicated to providing a harassment-free experience for participants at our event(s). `<EVENT>` is a conference intended for professional networking and collaboration in the R community. 
+
+We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. An inclusive space free of harassment encourages interaction among diverse groups. `<EVENT>` is likely the first encounter of many people with this community; therefore, we want to make certain the experience is welcoming, and encourages them to be involved moving forward.
```
Comment (hadley) [inline conduct/code-of-conduct.md]: My personal take is that you can't have a diverse and inclusive conference without first ensuring safety. And safety is all a CoC can really provide — we can protect against bad behaviour, but I think it's very difficult to enforce good behaviour.

Comment (sckott) [reply conduct/code-of-conduct.md]: Okay, what are you suggesing with respect to this suggested paragraph then?

Comment (hadley) [reply conduct/code-of-conduct.md]: No concrete suggestions 😄

Comment (karawoo) [reply conduct/code-of-conduct.md]: I don't really understand what "`<EVENT>` is likely the first encounter of many people with this community" means

Comment (sckott) [reply conduct/code-of-conduct.md]: do you mean what community? or does the whole sentence just not make sense?

Comment (karawoo) [reply conduct/code-of-conduct.md]: Yeah, I don't know what the community being referenced is, or how we know that this will be people's first encounter with it. And I think making the experience welcoming and encouraging is valuable regardless. Could we maybe rephrase to "We want to make certain `<EVENT>` is welcoming, and encourages participants to be involved moving forward."?

Comment (karawoo) [reply conduct/code-of-conduct.md]: @sckott are you open to the change I proposed? I think this is the last pending change, let me know if there is anything I can do to finish this off.

Comment (sckott) [reply conduct/code-of-conduct.md]: used your suggested language

File: conduct/code-of-conduct.md [hunk 2]
```diff
@@ -2,7 +2,9 @@ This is a template code of conduct that event organizers can use for conferences
 
 # Code of Conduct
 
-`<EVENT>` is dedicated to providing a harassment-free experience for participants at our event(s). `<EVENT>` is a conference intended for professional networking and collaboration in the R community. We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. 
+`<EVENT>` is dedicated to providing a harassment-free experience for participants at our event(s). `<EVENT>` is a conference intended for professional networking and collaboration in the R community. 
+
+We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. An inclusive space free of harassment encourages interaction among diverse groups. We want to make certain <EVENT> is welcoming, and encourages participants to be involved moving forward.
```
Comment (karawoo) [inline conduct/code-of-conduct.md]:
```suggestion
We encourage the open exchange of ideas and expression and thus require an environment that recognizes the inherent worth of every person and group. An inclusive space free of harassment encourages interaction among diverse groups. We want to make certain `<EVENT>` is welcoming, and encourages participants to be involved moving forward.
```

recommend putting `<EVENT>` in backticks for consistency with the rest of the doc

Comment (sckott) [reply conduct/code-of-conduct.md]: done
